test cases/frameworks/23 hotdoc/meson.build

```diff
@@ -7,3 +7,9 @@ endif
 
 subdir('doc')
 
+assert(hotdoc.has_extensions(['gi-extension']) == true,
+  'gi-extension is packaged with core, should always be present')
+
+assert(hotdoc.has_extensions(['gi-extension', 'no-way-you-exist-extension']) == false,
+  'gi-extension is packaged with core, should always be present')
```

Comment:
I don't understand the error message here :)

Reply:
Better?

Reply:
Yes, brb adding a `no-way-you-exist` extension